modules/nlu/src/backend/dialog-conditions/extracted-entity.ts

```diff
@@ -0,0 +1,53 @@
+import { Condition } from 'botpress/sdk'
+import _ from 'lodash'
+
+export default {
+  id: 'extracted_entity',
+  label: `An entity was extracted from the user's message`,
+  description: `Entity {type} {comparison} {value}`,
+  params: {
+    type: {
+      required: true,
+      label: 'Select the type of entity',
+      type: 'list',
+      list: {
+        endpoint: 'BOT_API_PATH/mod/nlu/entities?asOption=true',
+        valueField: 'value',
+        labelField: 'label'
+      }
+    },
+    comparison: {
+      label: 'Comparison method',
```

Comment:
it's not clear what you compare when you read this... entities count ? score ?

Reply:
The entity content itself. For example, in a single choice, you can type 1, 2 or 3 to select the first, second or third choice displayed. This adds one trigger per choice, so "if the value entity.number === 1"